doc/math.qbk

```diff
@@ -619,6 +620,7 @@ and as a CD ISBN 0-9504833-2-X  978-0-9504833-2-0, Classification 519.2-dc22.
 [include quaternion/math-quaternion.qbk]
 [include octonion/math-octonion.qbk]
 [include gcd/math-gcd.qbk]
+[include interpolators/cubic_b_spline.qbk]
```

Comment:
This seems to be a hangover from something else - that file should be included already (in a different location).